packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -113,6 +117,7 @@ export const AdvancedSettings = () => {
     )
   }, [
     initialDestinationAddressFromQueryParams,
+    isLoadingAccountType,
```

Comment:
This is not used in this useEffect hook, why is it added?

Reply:
i used it in some commits but deleted the lines and forgot to remove it from the dependency array~ thanks!